<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NouvelEvenementDialog</class>
 <widget class="QDialog" name="NouvelEvenementDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nouvel événement</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../application.qrc">
    <normaloff>:/assets/images/icon.png</normaloff>:/assets/images/icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout_3"/>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="mainWidgetEvent" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QFormLayout" name="formLayout">
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="2" column="0">
             <widget class="QRadioButton" name="radioButton_evenement">
              <property name="text">
               <string>Événement</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QRadioButton" name="radioButton_tache">
              <property name="text">
               <string>Tâche</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Titre</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="titre"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_3" native="true">
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Description</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QCalendarWidget" name="jour_debut"/>
            </item>
            <item row="4" column="1">
             <widget class="QCalendarWidget" name="jour_fin"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Jour de début</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QTimeEdit" name="heure_debut"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Heure de début</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Jour de fin</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QTimeEdit" name="heure_fin"/>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Heure de fin</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="description"/>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lieu"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Lieu</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listeTaches"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>radioButton_evenement</tabstop>
  <tabstop>radioButton_tache</tabstop>
  <tabstop>titre</tabstop>
  <tabstop>description</tabstop>
  <tabstop>lieu</tabstop>
  <tabstop>jour_debut</tabstop>
  <tabstop>heure_debut</tabstop>
  <tabstop>jour_fin</tabstop>
  <tabstop>heure_fin</tabstop>
  <tabstop>listeTaches</tabstop>
 </tabstops>
 <resources>
  <include location="../application.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NouvelEvenementDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>689</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NouvelEvenementDialog</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>415</x>
     <y>676</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_evenement</sender>
   <signal>toggled(bool)</signal>
   <receiver>NouvelEvenementDialog</receiver>
   <slot>toggleEvenementTache(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>toggleEvenementTache(bool)</slot>
  <slot>save()</slot>
 </slots>
</ui>
